index.html

```diff
@@ -126,6 +126,69 @@ <h2 class="text-center">Babel transforms your JavaScript</h2>
     </div>
   </div>
 
+    <hr class="featurette-divider">
+
+    <div class="row featurette">
+      <div class="col-md-6">
+        <h2>Polyfill</h2>
+        <p>Since Babel only transforms syntax (like arrow functions), you can use babel-polyfill in order to support new globals such as Promise or new native methods like String.padStart (left-pad). It uses <a href="https://github.com/zloirock/core-js">core-js</a> and <a href="http://facebook.github.io/regenerator/">regenerator</a>. Check out our <a href="/docs/usage/polyfill">babel-polyfill</a> docs for more info.</p>
+
+        <p>You can install the polyfill with</p>
+  {% highlight javascript %}
+  $ npm install --save-dev babel-polyfill
+  {% endhighlight %}
+        <p>Use it by requiring it at the top of the entry point to your application or in your bundler config.</p>
+      </div>
+
+      <div class="col-md-6">
+        <div class="col-md-6">
+          <ul class="babel-tick-list">
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer">ArrayBuffer</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from">Array.from</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/of">Array.of</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/copyWithin">Array.#copyWithin</a></li>
```

Comment:
This has still got the `.` which I don't think is common. .fill .find and.findIndex are also prototype methods